explorer/explorer.go

```diff
@@ -1565,6 +1567,16 @@ func (exp *Service) GetStateRootHash(blockHeight int64) (string, error) {
 	return hex.EncodeToString(rootHash[:]), nil
 }
 
+// GetMeta gets meta data of a blockchain
+func (exp *Service) GetMeta() (explorer.Meta, error) {
```

Comment:
we are migrating to new api, which is under /api/. Could you update it as well?
https://github.com/iotexproject/iotex-core/blob/master/proto/types/blockchain.proto#L74